src/main/java/mate/controller/LoginController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.exception.AuthenticationException;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final AuthenticationService authenticationService =
+            (AuthenticationService) injector.getInstance(AuthenticationService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String username = req.getParameter("login");
+        String password = req.getParameter("password");
+        try {
+            Driver user = authenticationService.login(username, password);
+            HttpSession session = req.getSession();
+            session.setAttribute("user_id", user.getId());
```

Comment:
<img width="797" alt="Screenshot 2021-12-14 at 12 05 11" src="https://user-images.githubusercontent.com/33026471/145978904-6cf8bbb3-3a02-494e-be22-d0359ed0c891.png">
let's add const for session attribute, fix it in all places